src/App.js

```diff
@@ -8,10 +8,10 @@ class App extends Component {
       <div className="App">
         <div className="App-header">
           <img src={logo} className="App-logo" alt="logo" />
-          <h2>Welcome to React</h2>
+          <h2>Welcome to my portfolio</h2>
         </div>
         <p className="App-intro">
-          To get started, edit <code>src/App.js</code> and save to reload.
+          Landing page will be here.
```

Comment:
At least following need to be fixed:
- [ ] landing page text
- [ ] "my portfolio" -> "MyPortfolio"
- [ ] animation timing